RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc

```diff
@@ -62,8 +62,8 @@ template <typename TrackerTraits>
 void SiPixelDigisClustersFromSoAT<TrackerTraits>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("src", edm::InputTag("siPixelDigisSoA"));
-  desc.add<int>("clusterThreshold_layer1", kSiPixelClusterThresholdsDefaultPhase1.layer1);
-  desc.add<int>("clusterThreshold_otherLayers", kSiPixelClusterThresholdsDefaultPhase1.otherLayers);
+  desc.add<int>("clusterThreshold_layer1", 2000);  //TODO put these somewhere
+  desc.add<int>("clusterThreshold_otherLayers", 4000);
```

Comment:
Please do not change from named constants to magic numbers :-)
Can you get their definition from somewhere ?

Reply:
Sure I can revert to the old ones. Still `kSiPixelClusterThresholdsDefaultPhase1.layer1` seems not much more informative that `clusterThreshold_layer1` : )

Reply:
Eh... the point is not whether `kSiPixelClusterThresholdsDefaultPhase1.layer1` is more or less descriptive than `clusterThreshold_layer1`.

What is useful is that `kSiPixelClusterThresholdsDefaultPhase1.layer1` is used consistently throughout the code to identify this value, instead of having `2000` appear in multiple places.

Reply:
You are right, I see the point. Redefining it somewhere sensible.